### PR TITLE
Prepare 3.1.0 final release #trivial

### PR DESCRIPTION
**Pull request by Kaspik**
- Update podspec file to final name 3.1.0
- Update Example project to use final version 3.1.0

### CODE REVIEW

File: Example/Podfile.lock [hunk 1]
```diff
@@ -3,7 +3,7 @@ PODS:
     - InputBarAccessoryView/Core (= 4.3.2)
   - InputBarAccessoryView/Core (4.3.2)
   - MessageKit (3.1.0):
-    - InputBarAccessoryView (~> 4.3.2)
+    - InputBarAccessoryView (~> 4.3.0)
```
Comment (austinwright) [inline Example/Podfile.lock:6]: We want this at 4.3.2. The swift package manager stuff is on `master` after the 4.3.2 tag.

Comment (Kaspik) [reply Example/Podfile.lock:6]: It points to 4.3.2 thanks to `~>` which means it's the highest `4.3.x`

Comment (Kaspik) [reply Example/Podfile.lock:6]: As you can see above that btw. :)

Comment (austinwright) [reply Example/Podfile.lock:6]: 🤦‍♂

**Review by austinwright — APPROVED** [review]
🎉

**Pull request merged**